test/unit/SwapOperator/swapEnzymeVaultShareForETH.js

```diff
@@ -0,0 +1,170 @@
+const { accounts, web3 } = require('hardhat');
+const { ether, expectEvent, expectRevert, time } = require('@openzeppelin/test-helpers');
+const { hex } = require('../utils').helpers;
+const { setNextBlockTime } = require('../utils').evm;
+const { assert } = require('chai');
+
+const [owner, governance, nobody] = accounts;
+const contracts = require('./setup').contracts;
+
+const { toBN } = web3.utils;
+const bnToNumber = bn => parseInt(bn.toString(), 10);
+const ETH = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE';
+
+const periodSize = 1800;
+const windowSize = 14400;
+
+const TIME_BETWEEN_SWAPS = time.duration.minutes(11);
+
+const nextWindowStartTime = async () => {
+  const now = bnToNumber(await time.latest());
+  const currentWindow = Math.floor(now / windowSize);
+  return (currentWindow + 1) * windowSize;
+};
+
+describe('swapEnzymeVaultShareForETH', function () {
```

Comment:
Might be worth checking for a revert when the time between swaps is less than MIN_TIME_BETWEEN_SWAPS.

Reply:
Related to maxSlippageRatio

we have this check that the minAmountOut param must be larger than slippage.

So you can set a min but it must be within slippage range.

https://github.com/NexusMutual/smart-contracts/blob/b2567a8971cf7d9c2c2cf493c70605dfd9cd87bc/contracts/modules/capital/SwapOperator.sol#L346